demisto_sdk/commands/validate/config_reader.py

```diff
@@ -42,18 +43,22 @@ def __init__(self, config_file_path=None, category_to_run=None):
             exit(1)
 
     def gather_validations_to_run(
-        self, use_git: bool, ignore_support_level: Optional[bool] = False
+        self, execution_mode: bool, ignore_support_level: Optional[bool] = False
```

Comment:
```suggestion
        self, execution_mode: executionMode, ignore_support_level: Optional[bool] = False
```
Don't foget to change the type as well.